NuGet.config

```diff
@@ -2,6 +2,7 @@
 <configuration>
   <packageSources>
     <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
+    <add key="AspNetCoreTools" value="https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json" />
```

Comment:
The reason is that the BuildTools are release branch invariant. If we have it pushed to a release \ branch specific feed, we have to ensure it gets primed prior to building which is a pain to deal with.